awx/sso/pipeline.py

```diff
@@ -187,13 +187,22 @@ def update_user_teams_by_saml_attr(backend, details, user=None, *args, **kwargs)
 
     team_ids = []
     for team_name_map in team_map.get('team_org_map', []):
-        team_name = team_name_map.get('team', '')
+        team_name = team_name_map.get('team', None)
+        team_alias = team_name_map.get('team_alias', None)
+        organization_name = team_name_map['organization']
```

Comment:
Should this be a `.get()`?  It's how it's access below on L195

Reply:
@ryanpetrello since the `organization` field is required, if you don't pass it on the configuration, the API will throw an error, so that key will exist.  But you are right, since I'm using `.get()` for team (which should also exist), let's keep it consistent and use `get()` here too.